kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/CustomResource.java

```diff
@@ -76,7 +76,7 @@
 public abstract class CustomResource<S, T> implements HasMetadata {
   private static final Logger LOG = LoggerFactory.getLogger(CustomResource.class);
 
-  private ObjectMeta metadata = new ObjectMeta();
+  protected ObjectMeta metadata = new ObjectMeta();
```

Comment:
This change breaks compatibility, but we are in `6.X` and, I think, it's a fair change ... @manusa

Reply:
Could you please elaborate on why this change is necessary

Reply:
it was needed to workaround this issue: https://github.com/sundrio/sundrio/issues/307
I should check, if it's completely fixed thanks to @iocanel I can revert this change.

Reply:
this is fixed